packages/usa-dark-background/src/styles/_usa-dark-background.scss

```diff
@@ -9,14 +9,6 @@
   }
 
   a {
-    color: color($theme-link-reverse-color);
-
-    &:visited {
-      color: color($theme-link-reverse-color);
-    }
-
-    &:hover {
-      color: color($theme-link-reverse-hover-color);
-    }
+    @include set-link-from-bg("base-darker");
```

Comment:
We could add `$theme-link-reverse-color` as the preferred link color for the mixin. This way it maintains the theme setting when appropriate but will resort to a fallback if need be

```suggestion
    @include set-link-from-bg("base-darker", $theme-link-reverse-color);
```
This results in a lighter link text than what's currently displayed

Current:
![Screenshot 2023-12-05 at 3 18 07 PM](https://github.com/uswds/uswds/assets/25211650/72ae82cc-9e99-4b15-9fda-408b3c57c0de)

With preferred theme token:

![Screenshot 2023-12-05 at 3 17 41 PM](https://github.com/uswds/uswds/assets/25211650/e69f3146-e4a9-424c-9ec0-7c6f40579e9a)

Reply:
@mahoneycm, I was unable to reproduce this. The mixin uses theme link colors by default and I was still getting `base-lighter` as the value (_for both passing and leaving out theme link reverse color_).

For clarity and consistency I've added:
1. A `$context` variable for future debugging.
2. Explicitly passing `$theme-link-reverse-color` to avoid any confusion.